c/include/ml-api-common.h

```diff
@@ -495,6 +495,86 @@ int ml_option_set (ml_option_h option, const char *key, void *value, ml_data_des
  */
 int ml_option_get (ml_option_h option, const char *key, void **value);
 
+/******************
+ * ML INFORMATION *
+ ******************/
+
+/**
+ * @brief A handle of a ml-information instance.
```

Comment:
Please define what is ml-information first, along with usage example as a guide in this header. (It's the data for training, fed-learning, ... . right?)

Reply:
@anyj0527 @myungjoo 
I updated PR #352 - usage of ml resource API using ml-information. Please review the update.